go/src/client/main.go

```diff
@@ -260,8 +260,18 @@ func playMatch(baselinePath string, candidatePath string, params []string, flip
 		case best_move := <-p.BestMove:
 			err := game.MoveStr(best_move)
 			if err != nil {
-				log.Println("Error decoding: " + best_move + " for game:\n" + game.String())
-				return 0, "", "", err
+				log.Println("Assuming resign: " + best_move + " for game:\n" + game.String())
+				//on white's turn it must be white resigning so black wins
+				if game.Position().Turn() == chess.White {
+					result = -1
+				} else {
+					result = 1
+				}
+				//if flip is set we have to flip result to report relative to candidate engine
+				if flip {
+					result = -result
+				}
+				break
```

Comment:
This break will apply to the select statement - you'll need to set a flag as well, and then conditionally break again, after the select is finished.